SDKBuildScripts/shared_build.sh

```diff
@@ -36,7 +36,9 @@ CleanCodeFiles () {
 BuildSdk () {
     pushd ..
     echo === SHARED BUILDING $SdkName ===
-    if [ -z "$targetSrc" ]; then
+    if [ ! -z "$SdkGenPrvTmplRepo" ]; then
+        node generate.js $SdkGenPrvTmplRepo=$destPath $apiSpecSource $SdkGenArgs $buildIdentifier $VerticalNameInternal
```

Comment:
>$SdkGenPrvTmplRepo [](start = 25, length = 18)

$destPath is where the generated SDK will be placed, right?

$SdkGenPrvTmplRepo sounds like "private template repo", i.e. a repo where the template resides.. I'm a bit confused.

Reply:
The "=" does not mean assignment, it's just clumsy legacy syntax.
Honestly, the discussion is lengthy, and will soon be redundant when we get genConfig.json online.
For now: It's not assignment, despite the appearance that it is. It's just crappy syntax that we'll remove soon.